engine/security/rootless.md

```diff
@@ -126,7 +126,7 @@ testuser::231072:65536
 
 ## Install
 
-The installation script is available at https://get.docker.com/rootless .
+The installation script is available at [here](https://get.docker.com/rootless).
```

Comment:
This would now read as "The installation script is available at here". Perhaps (given that this would likely be used on the command line, so I think we should not hide the actual URL). If we want the link to be clickable, I think we should also add a "target=blank", so that the script opens in a new tab/window instead of navigating the current page away from the documentation;

```suggestion
The installation script is available at [https://get.docker.com/rootless](https://get.docker.com/rootless){: target="_blank" class="_" }.
```